guide/13-managing-arcgis-applications/introduction_to_hub_initiatives_and_sites.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
Add a cell at the top that:
* is an individual markdown cell with nothing but a single hashmark followed by a title for the notebook